### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by MudasirSherwani**
I have worked in the branch:

- Delete boilerplate
- Add Calculator.js component
- Add CSS 
- Update README file
- Fix Linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@
 - [📝 License](#license)
```
Comment (omarsalem7) [inline README.md:25]: - [x] Kindly fix the broken MIT license link and add your own license.

File: src/component/Calculator.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  justify-content: center;
+  align-items: center;
+  background-color: #858694;
+  color: #fff;
+  padding-inline-end: 5px;
+  text-align: right;
+  padding: 17px;
+  font-size: 30px;
+  cursor: default;
+}
```
Comment (omarsalem7) [inline src/component/Calculator.css]: - [x] Kindly fix the design for the display part to match the provided design 

| Your design  | Provided design|
| ------------- | ------------- |
| image  | image |

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @MudasirSherwani 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️
- Correct git-flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — APPROVED** [review]
Hi @MudasirSherwani 👋, 
done-gif

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Proper use of Gitflow 👍 
* Well written PR description 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**